muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/sass/elements/application-panel.scss

```diff
@@ -122,6 +122,7 @@
     position: relative;
     right: 0;
     top: 0;
+    position: sticky;
```

Comment:
It seems we have 2 position properties. Wonder why Hound did not catch this. Also if `position: sticky` is the one to be used it needs to be placed before `right` property as they need to be placed in alphabetical order.